src/remediations/fifi.js

```diff
@@ -68,6 +69,32 @@ exports.sortSystems = function (systems, column = 'system_name', asc = true) {
     return _.orderBy(systems, column, (asc) ? 'asc' : 'desc');
 };
 
+
+// TODO: Replace this logic with logic in the remediations-consumer
+exports.updatePlaybookRunsStatus = async function (playbook_runs) {
+    _.forEach(playbook_runs, run => {
+        if (_.some(run.executors, executor => executor.status === FAILURE)) {
+            run.status = FAILURE;
```

Comment:
This logic seems to differ from what's done in https://github.com/RedHatInsights/remediations-consumer/blob/f40a63289dedff07c0776ca03bee6d3deef8798f/src/handlers/receptor/sharedQueries.ts#L28. Is that on purpose?

Reply:
Yes, this was on purpose.  Due to the fact that an RHC `playbook_run` will not be updated at the top level I figured I would include the logic for RUNNING and PENDING along with the 3 FINAL_STATES so that the status of the job makes more sense to the user.

Reply:
@jharting Gotcha!  Sorry I missed that.  This most recent push should have fixed the priority.